scheds/rust/scx_lavd/src/bpf/main.bpf.c

```diff
@@ -1749,9 +1754,11 @@ static void put_global_rq(struct task_struct *p, struct task_ctx *taskc,
 	}
 
 	/*
-	 * Enqueue the task to the eligible DSQ based on its virtual deadline.
+	 * Enqueue the task to one of the DSQs based on its virtual deadline.
 	 */
-	scx_bpf_dispatch_vtime(p, LAVD_GLOBAL_DSQ, LAVD_SLICE_UNDECIDED,
+	if (no_2_level_scheduling || is_lat_cri_task(taskc))
```

Comment:
I was a bit confused by this, but basically with `no_2_level_scheduling` all tasks are considered "latency critical" and they're all dispatched to the `LAVD_LATENCY_CRITICAL_DSQ`. Maybe it could be useful to add a small comment, so it can help the reader to immediately understand this logic.

Reply:
Thank you, @arighi, for the suggestion. I will update it as suggested.